_posts/2019-05-28-diamond.md

```diff
@@ -0,0 +1,11 @@
+---
+site: freiburg
+title: "Visit from Macromolecular Crystallography Group"
+tags: ???
+supporters:
+ - eosc
+---
```

Comment:
is this right? should I also mention them in the text?

Reply:
Yes, but maybe we need to add EOSC to the website-framework. Meaning adding a logo etc to the website.